src/fileformats/ocd_file_import.cpp

```diff
@@ -2051,23 +2048,33 @@ Symbol* OcdFileImport::getLayoutObjectSymbol(const O& ocd_object)
 					if (ok)
 						text_symbol->font_size = qRound(1000.0 * f_value / 72.0 * 25.4);
 					break;
-				case 'o':	// probably opacity value
+				case 'o':
+					f_value = param_value.toFloat(&ok);
+					if (ok && f_value >= 0.f && f_value <= 100.f)
+						opacity = 0.01f * f_value;
 					break;
 				default:
 					; // nothing
 				}
 			}
+			text_symbol->color = get_map_color(quint32(ocd_object.color), opacity);
 			text_symbol->updateQFont();
 			symbol = text_symbol;
 		}
 		break;
 	default:
-		addWarning(tr("Encountered an unsupported type of layout object (%1). Skipping.").arg(ocd_object.type));
+		addWarning(tr("Encountered an unsupported type of graphical object (%1). Skipping.").arg(ocd_object.type));
 		break;
 	}
 	
 	if (symbol)
 	{
+		if (ocd_object.symbol == Ocd::LayoutObject)
+			addWarningOnce(tr("Importing layout objects using auxiliary symbols. Export as layout objects is not possible."));
+		else if (ocd_object.symbol == Ocd::ImageObject)
+			addWarningOnce(tr("Importing image objects using auxiliary symbols. Export as image objects is not possible."));
+		else
+			addWarningOnce(tr("Importing graphic objects using auxiliary symbols. Export as graphic objects is not possible."));
```

Comment:
I very much like these warnings and thought about them some time ago. Can you please look into integrating the warning messages into the earlier patches that implement the respective functionality?